lib/utils/types.js

```diff
@@ -21,5 +21,12 @@ module.exports = {
   RENDERER: {
     SIMPLE: 'SIMPLE',
     ROTATING: 'ROTATING'
+  },
+
+  MARKER_TYPES: {
```

Comment:
Wouldn't it be more consistent if this was RED: 'RED' ? 
Just like the RENDERER above?